src/main/drivers/pwm_output.c

```diff
@@ -134,22 +134,12 @@ static void pwmWriteStandard(uint8_t index, uint16_t value)
 }
 
 #if defined(STM32F10X) && !defined(CC3D)
-static void pwmWriteOneshot125(uint8_t index, uint16_t value)
```

Comment:
Shouldn't need to be - I think it can be standardised.

```
#if defined(STM32F10X) && !defined(CC3D)
static void pwmWriteOneshot(uint8_t index, uint16_t value)
{
    *motors[index]->ccr = value * 7 / 6;
}
#else
static void pwmWriteOneshot(uint8_t index, uint16_t value)
{
    *motors[index]->ccr = value;
}
#endif
```
is the code in betaflight - so thats where its come from.